### DISCUSSION TITLE
Swipe to change languages

### DISCUSSION THREAD

**Discussion by BorodatiyKy** [florisboard/florisboard#2186] [Feedback]
Hey there!
 I've checked disscusion thread and didn't find the suggestion - to have an option to change languages  by swiping the spacebar, instead of pressing lang.button all the time.
Best wishes and thank you!~

**Comment by klaurence** [comment]
In `Settings > Gestures & Glide typing > Space bar gestures`, you should be able to select `Switch to previous subtype` for `Space bar swipe left`, and `Switch to next subtype` for `Space bar swipe right`. In Florisboard, keyboard layouts (whose languages you can modify) are called "subtypes".

**Reply by BorodatiyKy** [reply]
Thanks a lot! Appreciate you help, i was using Touchpal all the time but it stopped working on MIUI 14.....
